universal_transfer_operator/src/universal_transfer_operator/constants.py

```diff
@@ -12,20 +12,11 @@
 
 class Location(Enum):
     LOCAL = "local"
-    HTTP = "http"
-    HTTPS = "https"
     GS = "gs"  # Google Cloud Storage
-    GOOGLE_DRIVE = "gdrive"
     S3 = "s3"  # Amazon S3
-    WASB = "wasb"  # Azure Blob Storage
-    WASBS = "wasbs"  # Azure Blob Storage
-    POSTGRES = "postgres"
-    POSTGRESQL = "postgres"
     SQLITE = "sqlite"
-    DELTA = "delta"
     BIGQUERY = "bigquery"
     SNOWFLAKE = "snowflake"
```

Comment:
Why do we consider these databases as locations? Don't we support Postgres in UTO?